src/main/java/de/hysky/skyblocker/config/categories/GeneralCategory.java

```diff
@@ -261,6 +262,27 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
 								.build())
 						.build())
 
+				//Waypoints
+				.group(OptionGroup.createBuilder()
+						.name(Text.translatable("text.autoconfig.skyblocker.option.general.waypoints"))
+						.collapsed(true)
+						.option(Option.<Boolean>createBuilder()
```

Comment:
I don't think this option is used right now, so it should be commented out.

Reply:
I think we can just leave it here. It will be used in the future.